fight/fight.js

```diff
@@ -0,0 +1,40 @@
+const Soldier = require('./soldier');
+const Doctor = require('./doctor');
+const Heavy = require('./heavy');
+
+doc = new Doctor('Greg House');
+soldier = new Soldier('Jackie Chan');
+heavySoldier = new Heavy('John Wick');
+
+let round = 1, attacker, defender, looser, damage;
+while(heavySoldier.isAlive() && soldier.isAlive()){
+
+    if(round%2){
+        attacker = soldier;
+        defender = heavySoldier;
+    }else{
+        attacker = heavySoldier;
+        defender = soldier;
+    }
+
+    console.log("\n>>>>Round " + round + ":");
+    console.log("[Attacker stats: name " + attacker.getName() + ", health " + attacker.getHealth() + ", level " + attacker.getLevel()+", xp = "+attacker.getXP()+"]");
+    console.log("[Defender stats: name " + defender.getName() + ", health " + defender.getHealth() + ", level " + defender.getLevel()+", xp = "+defender.getXP()+"]");
+
+    if(round%3 === 0){
+        looser = attacker.getHealth() < defender.getHealth() ? attacker: defender;
+        console.log('Medic ' + doc.getName() + ' healed '+looser.getName()+' for ' + doc.heal(looser) + " hp");
+    }
+
+    if (attacker.hasOwnProperty('attackPower')) {
+        damage = attacker.handAttack(defender);
+    }else{
+        damage = attacker.machineGunAttack(defender, attacker.getLevel());

```

Comment:
Have no idea why noOfShots is here and how to use it correctly and not to kill the opponent with 1 shot :) as 15 shots with 15 damage is 225 damage! and only 100hp. so i changed it to level of shooter. Still don't get why can't we get it directly from this inside of function but need a param for that

Reply:
Hah, thanks for the feedback, we will fix this one in future :) Our system need some balance, I know